hack/verify-golint.sh

```diff
@@ -92,38 +89,21 @@ else
 			echo "$err"
 		done
 		echo
-		echo 'Please fix the above errors. You can test via "golint" and commit the result.'
+		echo 'Please review the above warnings. You can test via "golint" and commit the result.'
+		echo 'If the above warnings do not make sense, you can exempt this package from golint'
+		echo 'checking by adding it to  hack/.golint_failures (if your reviewer is okay with it).'
 		echo
 	} >&2
 	false
 fi
 
-# check to make sure all packages that pass lint are in the linted file.
```

Comment:
It doesn't look like we have the inverse check anymore.. to at least notify (if not fail) that a package in `.golint_failures` would actually pass golint and should be removed from the file.